README.md

```diff
@@ -7,135 +7,141 @@
+To open the workspace in Xcode, run the following command from the root directory of the repository:
 
-~~~
+```shell
 make open
-~~~
+```
 
-#### Command line integration
+### Command line integration
 
-You can run all the test suites from command line:
+To run all test suites from the command line, use the following command:
 
-~~~
+```shell
 make test
-~~~
+```
 
-### Code Style
+### Code style
 
-This project uses [SwiftLint](https://github.com/realm/SwiftLint) to check and enforce the Swift style and conventions. Style checks are automatically applied when the project is built from Xcode.
+This project uses [SwiftLint](https://github.com/realm/SwiftLint) to check and enforce Swift style and conventions. Style checks are automatically applied when the project is built from Xcode.
 
-To install the necessary tools and enable the Git pre-commit hook to autocorrect the style on each commit, run the following to update the project's git config `core.hooksPath`:
+To install the required tools and enable the Git pre-commit hook for automatic style correction on each commit, update the project's Git config `core.hooksPath` by running:
 
-~~~
+```shell
 make setup-tools
-~~~
+```
 
 ## Related Projects
 
-| Project                                                                              | Description                                                  |
-| ------------------------------------------------------------------------------------ | ------------------------------------------------------------ |
-| [AEPCore Extensions](https://github.com/adobe/aepsdk-core-ios)                       | The AEPCore and AEPServices represent the foundation of the Adobe Experience Platform SDK. |
-| [AEPConsent Extension](https://github.com/adobe/aepsdk-edgeconsent-ios)              | The AEPConsent extension enables consent preferences collection from your mobile app when using the AEP Mobile SDK and the Edge Network extension. |
-| [AEPLifecycle Extension](https://github.com/adobe/aepsdk-core-ios)                   | The AEPLifecycle extension helps collect application Lifecycle metrics and any additional context data provided by the application developer when using AEP SDK and the AEP Edge Network extension. |
-| [AEPEdgeIdentity Extension](https://github.com/adobe/aepsdk-edgeidentity-ios)        | The AEPEdgeIdentity extension enables handling of user identity data from a mobile app when using AEP SDK and the AEP Edge Network extension. |
-| [AEP SDK Sample App for iOS](https://github.com/adobe/aepsdk-sample-app-ios)         | Contains iOS sample apps for the AEP SDK. Apps are provided for both Objective-C and Swift implementations. |
-| [AEP SDK Sample App for Android](https://github.com/adobe/aepsdk-sample-app-android) | Contains Android sample app for the AEP SDK.                 |
+| Project | Latest Release | <img src="https://img.shields.io/badge/GitHub-%23000000.svg?logo=github&logoColor=white" height="24"/> |
```

Comment:
We should still avoid doc links which might change and this will be broken. Instead you can just have the github repos linked and remove the separate github section.